Examples/Tests/collision/analysis_collision_2d.py

```diff
@@ -51,8 +52,18 @@
 b = -0.083851393560288
 
 last_fn = sys.argv[1]
+# Remove trailing '/' from file name, if necessary
 if (last_fn[-1] == "/"): last_fn = last_fn[:-1]
-fn_list = glob(last_fn[:-5] + "?????")
+# Find last iteration in file name, such as 'test_name_plt000001' (last_it = '000001')
+last_it = re.search('\d+', last_fn).group()
```

Comment:
I think this may fail if there's a digit anywhere in `test_name_plt`. Perhaps search for `'\d+$'`?

Reply:
Thanks Phil, good idea! :)